### DISCUSSION TITLE
Issue sending emails with special character like á, é, í, ó, ú

### DISCUSSION THREAD

**Discussion by IvanCoto** [PHPMailer/PHPMailer#2689] [Q&A]
Hello all, this is my website [www.ci-technologycr.com](url)
The question is, how can i send emails with special characters, when a customer send me an email with special characters, for example "más información" in english is more information I receive it but, somthing like this, MÃ¡s informaciÃ³n.
I tried to copy this meta tag in my index.html, however it doesn't work. 
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">

**Comment by Synchro** [comment]
Add this to your script:

    $mail->CharSet = 'UTF-8';

and it will just work, assuming that all your code, HTML, and database are also using UTF-8, which they should be by default.

**Reply by IvanCoto** [reply]
Thank you very much! Now it is working.
